RELEASING.md

```diff
@@ -18,3 +18,5 @@ This page describes how to cut releases of our plugin. It's strictly for the own
 1. At the prompt set the version for release to {$version}
 1. The next prompt will ask you for the next snapshot release which you should set as {$nextVersion}-SNAPSHOT
 1. Once execution completes the tag and release should appear automatically on Github and Travis will trigger a deployment of the release artifacts to Github and the Jetbrains plugin repository.
+1. Push your release branch to Github, create a PR, and merge to master.
```

Comment:
I second @elharo's comment. The ordering of step 5 and 6 looks like it may need to be reversed - running a release locally seems like it should only be creating the tag and a commit. For the release to show up in Github and for travis to pick it up I'd think that 6 needs to happen first - but I haven't done a release myself yet on this project so I can't confirm.